test/integration/data.py

```diff
@@ -386,19 +386,19 @@ def _prepare_tableau_load(self, file_name, index_name, index_template):
 
 	def _post_ndjson(self, ndjsons, index_name, pipeline_name=None):
 		print("Indexing data for index '%s'." % index_name)
-		url = "%s/%s/_doc/_bulk" % (self._es.base_url(), index_name)
+		url = "%s/%s/_bulk" % (self._es.base_url(), index_name)
```

Comment:
Although removing `_type` from the control documents in the bulk request body is good in both 7.x and 8.x, I think `_doc` should only be removed from the URL in 8.x.

So this line might need changing in the backport to 7.x.

Please could you confirm @romseygeek?

Reply:
Typeless URLs work in 7x as well, so you shouldn't need to change anything.